recipes/velocypack/build.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -u
+ferr(){
+    echo "$@"
+    exit 1
+}
+
+CXX_STANDARD=${CXX_STANDARD:-14}
+echo "requesting c++ standard ${CXX_STANDARD}"
+BUILD_TYPE=${BUILD_TYPE:-Release}
```

Comment:
I am not sure you need this variable, since you don't use it in the cmake call below

Reply:
removed BUILD_TYPE=${BUILD_TYPE:-Release}
kept CXX_STANDARD used in the cmake call